### ISSUE TITLE
Sending a message with a variable in config.py gives an old value.

### ISSUE THREAD

**Issue by jancelin** [eternnoir/pyTelegramBotAPI#1412]
1. What version of pyTelegramBotAPI are you using?

```pyTelegramBotAPI 4.3.1```

2. What OS are you using?

```Debian GNU/Linux 11 (bullseye)```

3. What version of python are you using?

```Python 3.9.2```

Hello!

I'm trying to insert in a ```bot.reply_to(message, config.myvar)``` a variable stored in a ```config.py``` and which is regularly updated by another python def.
It works but when I call the bot it returns the default (original) value of the variable not the new one.
All my other services see the new value .... but not telebot

my code:

* mypython.py:

```
...
import telebot
import config

config.api_key = str( sys.argv[1] )
bot = telebot.TeleBot(config.api_key)

@bot.message_handler(commands=['mp'])
def send_mp(message):
    bot.reply_to(message, "Connected to: "+config.mp_use)

bot.infinity_polling()
```

* config.py

```
...

###defaut mountpoint
mp_use = "CT"

##Telegram
api_key = ''

...
```

**Comment by jancelin** [comment]
![photo_2022-01-13_13-23-23](https://user-images.githubusercontent.com/6421175/149329673-fbf1bc0c-5d0f-4916-899e-1a13585b47c9.jpg)

**Comment by jancelin** [comment]
I use python-telegram-bot for the first message

**Comment by Badiboy** [comment]
How this question relates to the pyTelegramBotAPI library?

**Comment by Badiboy** [comment]
> All my other services see the new value .... but not telebot

Print the value before use it with telebot and check what's printed.

**Comment by jancelin** [comment]
```
print(config.mp_use)

@bot.message_handler(commands=['mp'])
def send_mp(message):
    print("test!!!!:"+config.mp_use)
    bot.reply_to(message, "Connected to: "+config.mp_use)
```

```
caster_basevar | SLP
caster_basevar | test!!!!:CT
```

**Comment by Badiboy** [comment]
Fine. Seems you should ask print developers why they print "CT".

**Comment by Badiboy** [comment]
PS. In my personal oppinion you did absolutely noncence trying to **make mailbox between processes using python module**, but it's out of the scope.

**Comment by jancelin** [comment]
> 

I don't understand your answer...
